Project_Starting_Point/Client/Client.cpp

```diff
@@ -21,39 +21,41 @@ int main()
 	ClientSocket = socket(AF_INET, SOCK_STREAM, IPPROTO_TCP);
 	SvrAddr.sin_family = AF_INET;
 	SvrAddr.sin_port = htons(27001);
-	SvrAddr.sin_addr.s_addr = inet_addr("127.0.0.1");
-	connect(ClientSocket, (struct sockaddr*)&SvrAddr, sizeof(SvrAddr));
+	SvrAddr.sin_addr.s_addr = inet_addr("127.0.0.1"); 
+	connect(ClientSocket, (struct sockaddr*)&SvrAddr, sizeof(SvrAddr)); // connecting client with server
 
-	uiSize = GetSize();
-	for (unsigned int l = 0; l < uiSize; l++)
+	uiSize = GetSize(); // calling in-home function which returns size O(n)
+	for (unsigned int l = 0; l < uiSize; l++)  //O(n)
 	{
 		string strInput;
-		ifstream ifs("DataFile.txt");
-		for (unsigned int iStart = 0; iStart < l; iStart++)
+		ifstream ifs("DataFile.txt"); // opens ifstream object on File 
```

Comment:
We _could_ (but I'm not sure if we need to) note refactoring opportunities. For example, we could move this file opening outside of the loop.